code/datums/martial/krav_maga.dm

```diff
@@ -0,0 +1,171 @@
+	var/mob/living/carbon/human/H = owner
+	H.martial_art.streak = "leg_sweep"
+
+/datum/action/lung_punch//referred to internally as 'quick choke'
+	name = "Lung Punch - Delivers a strong punch just above the victim's abdomen, constraining the lungs. The victim will be unable to breathe for a short time."
+	button_icon_state = "lungpunch"
+
+/datum/action/lung_punch/Trigger()
+	if(owner.incapacitated())
+		owner << "<span class='warning'>You can't use Krav Maga while you're incapacitated.</span>"
+		return
+	owner << "<b><i>Your next attack will be a Lung Punch.</i></b>"
+	owner.visible_message("<span class='danger'>[owner] assumes the Lung Punch stance!</span>")
+	var/mob/living/carbon/human/H = owner
+	H.martial_art.streak = "quick_choke"//internal name for lung punch
+
+/datum/martial_art/krav_maga/teach(var/mob/living/carbon/human/H,var/make_temporary=0)
+	..()
+	H << "<span class = 'userdanger'>You know the arts of Krav Maga!</span>"
+	H << "<span class = 'danger'>Place your cursor over a move at the top of the screen to see what it does.</span>"
+	neckchop.Grant(H)
+	legsweep.Grant(H)
+	lungpunch.Grant(H)
+
+/datum/martial_art/krav_maga/remove(var/mob/living/carbon/human/H)
+	..()
+	H << "<span class = 'userdanger'>You suddenly forget the arts of Krav Maga...</span>"
+	neckchop.Remove(H)
+	legsweep.Remove(H)
+	lungpunch.Remove(H)
+
+/datum/martial_art/krav_maga/proc/check_streak(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	switch(streak)
+		if("neck_chop")
+			streak = ""
+			neck_chop(A,D)
+			return 1
+		if("leg_sweep")
+			streak = ""
+			leg_sweep(A,D)
+			return 1
+		if("quick_choke")//is actually lung punch
+			streak = ""
+			quick_choke(A,D)
+			return 1
+	return 0
+
+/datum/martial_art/krav_maga/proc/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.weakened)
+		return 0
+	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
+					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
+	D.apply_damage(5, BRUTE)
+	D.Weaken(4)//originally was 6, lowered since you could kill somebody in one stun
+	return 1
+
+/datum/martial_art/krav_maga/proc/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	D.losebreath += 5
+	D.adjustOxyLoss(10)
+	return 1
+
+/datum/martial_art/krav_maga/proc/neck_chop(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	D.visible_message("<span class='warning'>[A] karate chops [D]'s neck!</span>", \
+				  	"<span class='userdanger'>[A] karate chops your neck, rendering you unable to speak for a short time!</span>")
```

Comment:
'for a short time' really breaks the flow of this message and it'll sound pretty weird